src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/template.json

```diff
@@ -370,44 +361,44 @@
       ]
     },
     "logging": {
-      "displayName": "Extensions - Logging",
-      "description": "Includes Uno.Extensions.Logging",
+      "displayName": "Logging",
+      "description": "Configures logging support in the application",
       "type": "parameter",
       "datatype": "choice",
       "defaultValue": "serilog",
       "choices": [
         {
           "choice": "none",
-          "description": "Disables logging",
-          "displayName": "None"
+          "description": "Logging is configured for the application",
+          "displayName": "Basic"
         },
         {
           "choice": "default",
-          "description": "Enables basic logging to the console",
-          "displayName": "Default"
+          "description": "Logging to the console is configured",
+          "displayName": "Console"
```

Comment:
Reading these 2 descriptions I would be very confused as to what these mean/do.. and I'm the one that set it up 🤣 
Remember technically they will get the same Console logging either way. The default actually wires things up with DI to use throughout their app where the Core / Basic is getting some logging output from the framework.

Reply:
Yeh I was struggling to a) give these options a name and b) a sensible description

Reply:
Updated but I'm sure it could be reworked more